.typos.toml

```diff
@@ -0,0 +1,34 @@
+[default.extend-identifiers]
+AnserStyle = "AnserStyle"
+datas = "datas"
+impl_froms = "impl_froms"
+selfs = "selfs"
+strustS = "strustS"
+
+[default.extend-words]
+anser = "anser"
+ba = "ba"
+foor = "foor"
+fo = "fo"
+ket = "ket"
+makro = "makro"
+mis = "mis"
```

Comment:
"misuses", then shouldn't be necessary.

Reply:
it's generated in crates/ide-db/src/generated/lints.rs, can we edit it by hand?

![image](https://github.com/rust-lang/rust-analyzer/assets/71162630/179fde83-83a3-48fe-83dd-07ece1ca2d26)

Reply:
Right, I noticed that before and forgot about it :facepalm:.